moveit_py/src/moveit/moveit_py_utils/CMakeLists.txt

```diff
@@ -13,4 +13,4 @@ install(
   ARCHIVE DESTINATION lib
   RUNTIME DESTINATION bin)
 
-install(DIRECTORY include/ DESTINATION include)
+install(DIRECTORY include/ DESTINATION include/moveit_py)
```

Comment:
As the package name is `moveit_py_utils`, that should also be the install directory?